os/arch/arm/src/common/up_initialize.c

```diff
@@ -267,5 +267,10 @@ void up_initialize(void)
 	/* Initialize the L2 cache if present and selected */
 
 	up_l2ccinitialize();
+
+#ifdef CONFIG_KERNEL_TEST_FW
```

Comment:
The up_xxx is an architecture specific code. But this isn't. Calling location is not correct.

Reply:
Ok. I moved it to os/kernel/init/os_start.c